integrations/extensions/starter-kits/elasticsearch/README.md

```diff
@@ -0,0 +1,40 @@
+# Elasticsearch
+
+## Background 
+
+This is a starter kit for accessing the IBM Cloud Databases for Elasticsearch via a custom extension to IBM watsonx Assistant.
+The purpose of doing so is to allow a chatbot built in IBM watsonx Assistant to search for information in Elasticsearch 
+and show what it finds in the chat.  This starter kit uses the custom extensions feature in [the new IBM watsonx Assistant experience](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-watson-assistant-faqs#faqs-new-experience) 
+using actions.  Anyone with an older IBM watsonx Assistant instance based on the classic experience and dialogs will 
+need to upgrade to the new IBM watsonx Assistant experience before they can use the features described here.
+
+## Install and set up Elasticsearch
+
+Before starting, you will need to provision a Elasticsearch instance and set up a search index. Please follow the steps in 
+[elasticsearch-install-and-setup-guide](../../docs/elasticsearch-install-and-setup/ICD_Elasticsearch_install_and_setup.md) for more details.
```

Comment:
OpenAPI spec should be in the /elasticsearch folder, not the /elasticsearch/assets folder for consistency with other search kits.